cli/cmd/plugin/ui/web/tanzu-ui/src/shared/constants/defaults/azure.defaults.tsx

```diff
@@ -25,11 +25,130 @@ export const AZURE_DEFAULT_VALUES = {
+    'Standard_DC4s_v2',
+    'Standard_DC2s_v3',
+    'Standard_DC1s_v3',
+    'Standard_DC4s_v3',
+    'Standard_DC2ds_v3',
+    'Standard_DC1ds_v3',
+    'Standard_DC4ds_v3',
+    'Standard_D2_v2',
+    'Standard_D1_v2',
+    'Standard_D3_v2',
+    'Standard_D2_v4',
+    'Standard_D4_v4',
+    'Standard_D2s_v4',
+    'Standard_D4s_v4',
+    'Standard_D2a_v4',
+    'Standard_D4a_v4',
+    'Standard_D2as_v4',
+    'Standard_D4as_v4',
+    'Standard_D2d_v4',
+    'Standard_D4d_v4',
+    'Standard_D2ds_v4',
+    'Standard_D4ds_v4',
+    'Standard_D2_v5',
+    'Standard_D4_v5',
+    'Standard_D2s_v5',
+    'Standard_D4s_v5',
+    'Standard_D2d_v5',
+    'Standard_D4d_v5',
+    'Standard_D2ds_v5',
+    'Standard_D4ds_v5',
+    'Standard_D2as_v5',
+    'Standard_D4as_v5',
+    'Standard_D2ads_v5',
+    'Standard_D4ads_v5',
+    'Standard_DC2as_v5',
+    'Standard_DC4as_v5',
+    'Standard_DC2ads_v5',
+    'Standard_DC4ads_v5',
+];
 
-const AZURE_DEFAULT_INSTANCE_TYPES: KeyOfStringToString = {
-    [AZURE_NODE_PROFILE_NAMES.SINGLE_NODE]: 'Standard_D2s_v3',
-    [AZURE_NODE_PROFILE_NAMES.HIGH_AVAILABILITY]: 'Standard_D2s_v3',
-    [AZURE_NODE_PROFILE_NAMES.PRODUCTION_READY]: 'Standard_D4s_v3',
+const PRODUCTION_READY_FALL_BACK: Array<string> = [
+    'Standard_D2s_v3',
+    'Standard_D8s_v3',
+    'Standard_D4_v3',
+    'Standard_D2_v3',
+    'Standard_D8_v3',
+    'Standard_A4_v2',
+    'Standard_A2_v2',
+    'Standard_A8_v2',
+    'Standard_B4ms',
+    'Standard_B2ms',
+    'Standard_B8ms',
+    'Standard_DC4s_v2',
+    'Standard_DC2s_v2',
+    'Standard_DC8_v2',
+    'Standard_DC4s_v3',
+    'Standard_DC2s_v3',
+    'Standard_DC8s_v3',
+    'Standard_DC4ds_v3',
+    'Standard_DC2ds_v3',
+    'Standard_DC8ds_v3',
+    'Standard_D4_v2',
+    'Standard_D3_v2',
+    'Standard_D5_v2',
+    'Standard_DS4_v2',
+    'Standard_DS3_v2',
+    'Standard_DS5_v2',
+    'Standard_D4_v4',
+    'Standard_D2_v4',
+    'Standard_D8_v4',
+    'Standard_D4s_v4',
+    'Standard_D2s_v4',
+    'Standard_D8s_v4',
+    'Standard_D4a_v4',
+    'Standard_D2a_v4',
+    'Standard_D8a_v4',
+    'Standard_D4as_v4',
+    'Standard_D2as_v4',
+    'Standard_D8as_v4',
+    'Standard_D4d_v4',
+    'Standard_D2d_v4',
+    'Standard_D8d_v4',
+    'Standard_D4ds_v4',
+    'Standard_D2ds_v4',
+    'Standard_D8ds_v4',
+    'Standard_D4_v5',
+    'Standard_D2_v5',
+    'Standard_D8_v5',
+    'Standard_D4s_v5',
+    'Standard_D2s_v5',
+    'Standard_D8s_v5',
+    'Standard_D4d_v5',
+    'Standard_D2d_v5',
+    'Standard_D8d_v5',
+    'Standard_D4as_v5',
+    'Standard_D2as_v5',
+    'Standard_D8as_v5',
+    'Standard_D4ads_v5',
+    'Standard_D2ads_v5',
+    'Standard_D8ads_v5',
+    'Standard_DC4as_v5',
+    'Standard_DC2as_v5',
+    'Standard_DC8as_v5',
+    'Standard_DC4ads_v5',
+    'Standard_DC2ads_v5',
+    'Standard_DC8ads_v5',
+];
+
+const AZURE_DEFAULT_INSTANCE_TYPES: KeyOfStringToArray = {
+    [AZURE_NODE_PROFILE_NAMES.SINGLE_NODE]: ['Standard_D2s_v3', ...SINGLE_NODE_FALL_BACK],
```

Comment:
how about just adding the 'Standard_D2s_v3' to the top of the list so it's not hard-coded here by itself?

Reply:
+1 I thought that was being done but didn't look close enough. Should do the same for production ready